go/runtime/flow_consumer.go

```diff
@@ -29,7 +29,7 @@ type FlowConsumerConfig struct {
 	Flow struct {
 		BuildsRoot string `long:"builds-root" required:"true" env:"BUILDS_ROOT" description:"Base URL for fetching Flow catalog builds"`
 		BrokerRoot string `long:"broker-root" required:"true" env:"BROKER_ROOT" default:"/gazette/cluster" description:"Broker Etcd base prefix"`
-		Network    string `long:"network" default:"host" description:"The Docker network that connector containers are given access to."`
+		Network    string `long:"network" description:"The Docker network that connector containers are given access to, defaults to none"`
```

Comment:
comment is misleading, as "none" means "doesn't have anything but loopback".

Perhaps reword to "defaults to creating a new docker bridge network"?